model/riscv_insts_base.sail

```diff
@@ -706,6 +833,17 @@ mapping clause assembly = FENCE(pred, succ)
   <-> "fence" ^ spc() ^ fence_bits(pred) ^ sep() ^ fence_bits(succ)
 
 /* ****************************************************************** */
+
+$[name fence (total store order)]
```

Comment:
Could you rectify this to capitalize "Fence"? You did what I originally asked, I see, but I asked for "fence" when I meant "Fence". My apologies.

Reply:
Oh, I should have noticed the consistency, Missed it. I have corrected it now.